src/sbml2symbolics.jl

```diff
@@ -0,0 +1,31 @@
+using ModelingToolkit, Symbolics
+
+allowed_funs = Dict(
+    "+" => :+,
+    "-" => :-,
+    "*" => :*,
+    "/" => :/,
+    #TODO add translations of SBML functions to Julia
```

Comment:
TODO

Reply:
I think this is best done when testing with the [sbml_test_suite](https://github.com/sbmlteam/sbml-test-suite/tree/master/cases/semantic). Then we see where the test cases fail and add translations on a case by case basis.